recipes/lunarcalendar/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "LunarCalendar" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 681142f22fc353c3abca4b25699e3d1aa7083ad1c268dce36ba297eca04bed5a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
You probably only need python and pip here

Reply:
I had initially tried that, but the problem is that `setup.py` has:
```
from lunarcalendar import __version__
```
which causes it to import everything in `lunarcaledar/__init__.py` which requires all of the run dependencies.